### ISSUE TITLE
arrows not shown for directed relationships in Neovis.js

### ISSUE THREAD

**Issue by lt21811** [neo4j-contrib/neovis.js#68]
All relationships in my graphs in neovis.js do not show the arrows (for directed relationships). Is there a setting that needs to be enabled?

**Comment by thebestnom** [comment]
yes, arrows, you can see all configuration in the readme, also you can see it in the index.d.ts

**Comment by lt21811** [comment]
@thebestnom Found it. Thank you!

**Comment by pg98gh** [comment]
Hi, 
i´m new to neo4j and neovis.js and found this conversation. I´m also having an issue with the arrows: If my query is not limited, the arrows and relationships are not shown. If i limit it to 20, it all works fine. 
@thebestnom do you have a hint for me, how to solve this issue? I´d very appreciate it.
Best regards

**Comment by thebestnom** [comment]
What version are you using?

**Comment by pg98gh** [comment]
I´m using 1.5.0 as i got the link for the screencast from a colleague and sticked to the code the neo4j team published on github. 
I also opened a ticket on stackoverflow where you can see my code: https://stackoverflow.com/questions/68890871/arrows-are-not-shown-if-limit-is-above-20

**Comment by thebestnom** [comment]
Hmm... that's really weird, Ill have to test that to see the problem 😅

**Comment by pg98gh** [comment]
> 
> 
> Hmm... that's really weird, Ill have to test that to see the problem 😅

ok, no Problem i appreciate your help :). I´m looking also for the solution now for several days but i still couldn´t come up with an answer :D

**Comment by thebestnom** [comment]
I would also suggest trying 2.0.0 alpha, it uses a newer version of vis.js, maybe it is a bug there (there is a migration guide on the branch 😅)

**Comment by pg98gh** [comment]
Thanks, for the hint I already implemented it, but the arrows and now the title_properties are still not shown. Here´s my code and i adjustes it to the 2.0.0 alpha but i´m still stuck somehow...
`function draw() {{
                        var config = {{
                            container_id: "viz",
                            neo4j:{{
                                server_url: "{uri_neo4j}",
                                server_user: "{user_neo4j}",
                                server_password: "{pwd_neo4j}",
                            }},
                            
                            labels: {{
                                FMEA:{{
                                    label:"clean_text",
                                    value:"pagerank",
                                    group:"community"                             
                                }}           
                            }},
                            relationships: {{
                                "CAUSES":{{
                                    value: "PID",
                                    label: true,
                                                                        
                                }},
                                "INCLUDES":{{ 
                                    value: "PID",
                                    label: true
                                    
                                }}
                            }},
                            arrows:true,
                            initial_cypher: "{query}"
                            
                        }};
                        var viz = new NeoVis.default(config);
                        viz.render();
                        }}
                </script>
                <body onload="draw()">
                    <div id="viz"></div>
                </body> `

**Comment by thebestnom** [comment]
In 2.0.0 how you add arrows changed (hence major breaking changes) but in the guide mention arrow specifically https://github.com/neo4j-contrib/neovis.js/blob/2.0.0/MIGRATING.md

**Comment by thebestnom** [comment]
Trivial in 2.x
